Assets/GothicVR/Scripts/Phoenix/Interface/PhoenixBridge.cs

```diff
@@ -18,5 +19,6 @@ public static class PhoenixBridge
 
         // FIXME Find a better place for the NPC routines. E.g. on the NPCs itself? But we e.g. need to have a static NPCObject List to do so.
         public static Dictionary<IntPtr, List<RoutineData>> npcRoutines = new();
+        public static Dictionary<IntPtr, Dictionary<DateTime, RoutineData>> npcRoutinesDict = new();
```

Comment:
Do we use this? Or: Can we get rid of it? The data is already inside npcObj.GetComponent<Routine>()

Reply:
yep, obsolete - deleted